public/i18n.js

```diff
@@ -1,3 +1,5 @@
+// NOTE: Trailing commas in this object are fine, per ECMAScript 5 (circa 2009).
```

Comment:
extraneous comment. Let's assume folks know the language.

Reply:
I felt it was worth explaining because the trailing commas were added by me. Otherwise instead of a single line you are modifying two lines.